<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogYear</class>
 <widget class="QDialog" name="DialogYear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить год</string>
  </property>
  <widget class="QLabel" name="lab_year">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_year">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_years">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Года</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_years">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
